apps/glusterfs/device_entry.go

```diff
@@ -398,6 +398,8 @@ func (d *DeviceEntry) Remove(db wdb.DB,
 	executor executors.Executor,
 	allocator Allocator) (e error) {
 
+	var ErrBrickWithEmptyPath error = fmt.Errorf("Brick has no path")
```

Comment:
This should probably be a file level constant. If its not meant to be used outside of the scope of this file just lowercase it.